tests/boot/boot_from_pxe.pm

```diff
@@ -31,7 +31,7 @@ sub run {
     if (check_var('BACKEND', 'ipmi')) {
         select_console 'sol', await_console => 0;
     }
-    assert_screen([qw(virttest-pxe-menu qa-net-selection prague-pxe-menu pxe-menu)], 300);
+    assert_screen([qw(virttest-pxe-menu qa-net-selection prague-pxe-menu pxe-menu orthos-pxe-menu)], 300);
```

Comment:
why do you add a new tag? I do not see this tag being used e.g. in a `if (match_has_tag('orthos-pxe-menu'))` so maybe just create needles with the tag "pxe-menu"?

Reply:
I will keep the new needle, and will employ match_has_tag rather than making decsion out of ARCH.